src/api/api.ts

```diff
@@ -0,0 +1,69 @@
+import { fetch } from "cross-fetch";
+import { type Result, Ok, Err } from "../types";
+import { hashSigningKey } from "../helpers/strings";
+import {
+  ErrorSchema,
+  type ErrorResponse,
+  StepsSchema,
+  type StepsResponse,
+  BatchSchema,
+  type BatchResponse,
+} from "./schema";
+
+interface InngestAPIConstructorOpts {
+  baseUrl?: string;
+  signingKey: string;
+}
+
+export class InngestAPI {
```

Comment:
The biggest of all the nits: Can this (and other related types) be `InngestApi` for consistency?